lib/app/modules/LoginView/views/login_view_view.dart

```diff
@@ -0,0 +1,182 @@
+                          controller.passwordVisible.value ? Icons.visibility : Icons.visibility_off,
+                        ),
+                        onPressed: () {
+                          controller.togglePasswordVisibility();
+                        },
+                      ),
+                    ),
+                    style: const TextStyle(color: Colors.white),
+                  ),
+                )),
+                const SizedBox(height: 20),
+                // Forgot Password
+                const Align(
+                  alignment: Alignment.centerRight,
+                  child: Text(
+                    'Forgot Password?',
+                    style: TextStyle(color: Colors.grey),
+                  ),
+                ),
+                const SizedBox(height: 20),
+                // Dropdown for Committee, Domain, Team with Shadow and Custom Dropdown Color
+                Obx(() => Container(
+                  decoration: BoxDecoration(
+                    color: theme.colorLevel1,
+                    borderRadius: BorderRadius.circular(theme.kTextFieldBorderRadius),
+                    boxShadow: [
+                      BoxShadow(
+                        color: Colors.grey.withOpacity(0.5),
+                        spreadRadius: 2,
+                        blurRadius: 5,
+                        offset: const Offset(0, 3),
+                      ),
+                    ],
+                  ),
+                  child: DropdownButtonFormField<String>(
+                    value: controller.selectedOption.value.isEmpty ? null : controller.selectedOption.value,
+                    dropdownColor: theme.colorLevel1,
+                    hint: const Text(
+                      'Select Committee, Domain, or Team',
+                      style: TextStyle(color: Colors.grey),
+                    ),
+                    decoration: InputDecoration(
+                      filled: true,
+                      fillColor: Colors.transparent,
+                      border: OutlineInputBorder(
+                        borderRadius: BorderRadius.circular(40.0),
+                        borderSide: BorderSide.none,
+                      ),
+                    ),
+                    items: controller.options.map((String option) {
+                      return DropdownMenuItem<String>(
+                        value: option,
+                        child: Text(option, style: const TextStyle(color: Colors.white)),
+                      );
+                    }).toList(),
+                    onChanged: (String? newValue) {
+                      controller.setSelectedOption(newValue);
+                    },
+                  ),
+                )),
+                const SizedBox(height: 40),
+                // Login Button
+                ElevatedButton(
+                  onPressed: () {},
+                  style: ElevatedButton.styleFrom(
+                    backgroundColor: Colors.transparent,
```

Comment:
add validation for textfields, means the login button should not work when fields are empty and should display required message (*Hint- You can use TextFormField to achieve this* ).